tck/src/main/java/org/apache/jdo/tck/util/ResultSummary.java

```diff
@@ -36,9 +37,14 @@ public class ResultSummary implements Serializable {
   /** The name of the file to store a serialized instance of this class. */
   private static final String FILE_NAME_OF_RESULT_SUMMARY = "ResultSummary.ser";
 
+  private static final DecimalFormat THREE_DIGITS_FORMATTER = new DecimalFormat("000");
```

Comment:
What are these for? They don't appear to be used...?

Reply:
The two constants are leftovers. I remove them.